wellregistry/registry/admin.py

```diff
@@ -27,40 +38,31 @@ def clean(self):
 
         if cleaned_data.get('well_depth') and not cleaned_data.get('well_depth_units'):
             raise forms.ValidationError(
-                 'If Well depth is populated, then you must enter a Well depth unit')
+                'If Well depth is populated, then you must enter a Well depth unit')
```

Comment:
These changes in the clean method are just reformatting. pylint was flagging a bunch of these. I really recommend that make runlint is run before submitting a PR. I found lots of things in the changes that I made to that needed to be improved.

Reply:
Thanks for the comment.  I'll keep that in mind!